docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+Try Trino with Docker
+======================
+
+This guide focuses on your first ten minutes
+using Trino, and relies on Docker to simplify the installation and
+ensure that the CLI runs on your platform of choice.
+
+This Docker container includes both the Trino server and the 
+Trino command line interface (CLI), which provides a terminal-based,
+interactive shell for running queries and inspecting catalog structures
+in large data clusters. In production, it's unlikely that your server 
+and client will reside in the same location. However, to keep this guide 
+short and sweet, we are going to run both from your machine.
+
+In this guide, you'll learn how to:
+
+-  Install and run the Trino Interactive CLI
+-  Call Trino from the terminal
+
+Prerequisites
+-------------
+
+Trino runs best on Linux. Our Docker container is built on 
+Linux, which means that you can run this guide from Linux, 
+macOS, or Windows.
+
+-  `Install Docker <https://docs.docker.com/get-docker/>`__
+
+Pull Trino from Docker
+------------------------------
+
+Once Docker is installed, the next step is to pull the Trino container
+from Docker. This container contains both the server and the CLI.
+
+From your terminal, run:
+
+.. code-block:: shell
+
+    docker pull trinodb/trino
+
+Start the Trino server
+----------------------
+
+After you've pulled the container, you'll need to start your Trino server. 
+You're going to make calls against this server (and it's data) throughout 
+this guide. 
+
+.. code-block:: shell
+
+   docker run -p 8080:8080 --name trino trinodb/trino
+
+When the server is ready, you'll see this message:
+
+.. code-block:: text
+
+   INFO    main    io.trino.server.Server    ======== SERVER STARTED ========
+
+Start the Trino CLI
+--------------------
+
+In a new terminal window or tab, run this command to start the Trino
+Interactive CLI:
+
+.. code-block:: shell
+
+    docker exec -it trino trino
+
+When the Trino Interactive CLI is ready to use, you should see:
```

Comment:
remove "Interactive" 
replace "you should see" with "look for".
(The only purpose of "you should" is to nag. That's not our job.)